datasets/lt/pep_declarations/crawler.py

```diff
@@ -0,0 +1,204 @@
+                },
+                cache_days=30,
+            )
+        except HTTPError as ex:
+            response = ex.response.json()
+            if status_code := response.pop("status") == 404:
+                self.context.log.info(f"deklaracija {id_str} does not exist")
+            else:
+                self.context.log.error(
+                    f"deklaracija {id_str} skipped due to {status_code} error."
+                )
+
+
+def make_person(context: Context, data: dict) -> Entity:
+    person = context.make("Person")
+    first_name = data.pop("vardas")
+    last_name = data.pop("pavarde")
+    person_id: Optional[str] = data.pop(
+        "asmensKodas", None
+    )  # this identifier is often missing
+    birth_date: Optional[str] = data.pop("gimimoData", None)
+    person.id = context.make_id(person_id, first_name, last_name)
+    person.add("firstName", first_name)
+    person.add("registrationNumber", person_id)
+    person.add("lastName", last_name)
+    person.add("birthDate", birth_date)
+    person.add("legalForm", data.pop("asmensTipas", None))
+    context.audit_data(data)
+    return person
+
+
+def make_spouse(context: Context, person: Entity, spouse: Entity) -> Entity:
+    relationship = context.make("Family")
+    relationship.id = context.make_id(person.id, spouse.id)
+    relationship.add(
+        "relationship", "Sutuoktinis, sugyventinis ar partneris", lang="lit"
+    )
+    relationship.add("person", person)
+    relationship.add("relative", spouse)
+    return relationship
+
+
+def parse_affiliations(
+    context: Context, person: Entity, affiliations: list[dict], is_pep: bool = True
+) -> list[tuple[Entity]]:
+    """
+    Args:
+        context (Context)
+        affiliations (list[dict]): A list of dicts, where each dict refers to a place
+         of work and a sub-list of positions.
+        is_pep (bool, optional): Defaults to True.
+
+    Returns:
+        list[tuple[Entity]]: a flattened list of tuples, where each is
+        a position and occupancy. Occupancies that do not meet OpenSanctions criteria
+        are skipped.
+    """
+    entities = []
+    for data in affiliations:
+        entity_is_legal_entity: bool = data.pop("yraJuridinisAsmuo")
+        entity_is_lithuanian: bool = data.pop("registruotaLietuvoje")
+        legal_entity_code: Optional[str] = data.pop("jaKodas")
+        entity_name = data.pop("pavadinimas")
+        affiliation_start_date: str = data.pop("rysioPradzia")
+        affiliation_type = data.pop("darbovietesTipas")
+        data_sources = data.pop("duomenuSaltiniai")
+        filed_automatically: bool = data.pop("uzpildytaAutomatiskai")
+        entity_must_be_declared: bool = data.pop("privaluDeklaruoti")
+        entity_legal_form: str = data.pop("jaTeisinesFormosPavadinimas")
+
+        for role in data.pop("pareigos"):
+            position_name: str = role.pop("pareigos") or "Unknown position"
```

Comment:
just a note here - not a priority, but we could make https://www.opensanctions.org/reference/#schema.Ownership's of Savininkė/Savininkas and https://www.opensanctions.org/reference/#schema.Directorship out of Direktorius/Directorė where entity_must_be_declared=False

Reply:
let's park this idea for now